src/dymaptic.GeoBlazor.Core/Components/Symbols/Symbol.cs

```diff
@@ -131,7 +132,7 @@ public SymbolSerializationRecord()
     public MapColor? HaloColor { get; init; }
 
     [ProtoMember(13)]
-    public int? HaloSize { get; init; }
+    public double? HaloSize { get; init; }
```

Comment:
is there a reason these are doubles, but in TextSymbol they are Dimension?

Reply:
Yes, good question. This is a protobuf serialization class, and I decided to handle all the conversions within the C#, not expose them to the serialization or JS.